functions/helpers/chatChannelJanitor.private.ts

```diff
@@ -44,7 +44,7 @@ const deleteProxySession = async (context: Context<EnvVars>, proxySession: strin
 
     if (!ps) {
       // eslint-disable-next-line no-console
-      console.warn(`Tried to remove proxy session ${proxySession} but couldn't find it.`);
+      console.log(`Tried to remove proxy session ${proxySession} but couldn't find it.`);
```

Comment:
Minor nit - I would use `console.info` rather than `console.log` - just to be super explicit and make it easier to migrate code to a logging framework and need to give every statement a log level